maestro-cli/src/main/java/maestro/cli/App.kt

```diff
@@ -63,10 +64,13 @@ class App {
     @Option(names = ["--host"], hidden = true)
     var host: String? = null
 
+    @Option(names = ["--driver-host-port"], hidden = true)
+    var driverHostPort: Int = Random.nextInt(9001, 9128)
+
     @Option(names = ["--port"], hidden = true)
     var port: Int? = null
 
-    @Option(names = ["--device", "--udid"], description = ["(Optional) Select a device to run on explicitly"])
+    @Option(names = ["--device", "--udid"], description = ["(Optional) Device ID to run on explicitly, can be a comma separated list of IDs: --udid \"Emulator_1,Emulator_2\" "])
```

Comment:
udid is specifically used because its the term that Apple uses it, so the example being Emulators (as opposed to Simulators) is unexpected

```suggestion
    @Option(names = ["--device", "--udid"], description = ["(Optional) Device ID to run on explicitly, can be a comma separated list of IDs: --device \"Emulator_1,Emulator_2\" "])
```